ci/run-experimental-x64-ci.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+cargo +nightly \
+            -Zfeatures=all -Zpackage-features \
+            test \
+            --features test-programs/test_programs \
+            --features experimental_x64 \
+            --all \
+            --exclude wasmtime-lightbeam \
+            --exclude peepmatic \
+            --exclude peepmatic-automata \
+            --exclude peepmatic-fuzzing \
+            --exclude peepmatic-macro \
+            --exclude peepmatic-runtime \
+            --exclude peepmatic-test --exclude lightbeam
```

Comment:
```suggestion
            --exclude peepmatic-test \
            --exclude peepmatic-souper \
            --exclude lightbeam
```

I ran this and realized that there is an additional crate to ignore to avoid running into issues if `z3` (required by peepmatic) is not installed locally.